spec/services/losing_bidder_email_sender_spec.rb

```diff
@@ -10,12 +10,12 @@
           losing_bid = create(:bid, amount: 100, auction: auction)
           second_losing_bid = create(:bid, amount: 101, auction: auction)
           mailer_double = double(deliver_later: true)
-          allow(AuctionMailer).to receive(:losing_bidder_notification).
-            with(losing_bid).
-            and_return(mailer_double)
-          allow(AuctionMailer).to receive(:losing_bidder_notification).
-            with(second_losing_bid).
-            and_return(mailer_double)
+          allow(AuctionMailer).to receive(:losing_bidder_notification)
+            .with(losing_bid)
```

Comment:
oh interesting I usually prefer the `.` at the ends of lines --- but whatever works!